src/explorer/ConnectionsTreeItem.ts

```diff
@@ -9,13 +9,14 @@ import { IAzureNode, IAzureParentTreeItem, IAzureTreeItem } from 'vscode-azureex
 import { CosmosDBTreeItem } from './CosmosDBTreeItem';
 
 export class ConnectionsTreeItem implements IAzureParentTreeItem {
-    public static contextValue: string = 'Connections';
+    public static contextValue: string = 'сonnectionsTreeItem';
```

Comment:
I liked the old contextValue better. We could add 'TreeItem' to every single context value if we wanted and it would just get out of hand